platform-includes/enriching-events/add-attachment/python.mdx

```diff
@@ -1,28 +1,27 @@
-Attachments live on the `Scope` and will be sent with all events.
+Attachments live on the [`Scope`](/platforms/python/enriching-events/scopes/). The SDK sends all attachments on a given `Scope` with every non-transaction event captured within the `Scope`. Optionally, an attachment can be sent with all events, including transactions, when the attachment is added with the `add_to_transactions` option set to `True`.
```

Comment:
```suggestion
Because attachments live on the [`Scope`](/platforms/python/enriching-events/scopes/), the SDK sends all attachments on a given `Scope` with every non-transaction event captured within the `Scope`. It's also possible to send an attachment with all events, including transactions by setting the `add_to_transactions` option to `True`.
```